docs/content/reference/routing-configuration/other-providers/docker.md

```diff
@@ -688,6 +688,16 @@ You can tell Traefik to consider (or not) the container by setting `traefik.enab
 
 This option overrides the value of `exposedByDefault`.
 
+#### `traefik.docker.visiblewhennotrunning`
+
+```yaml
+- "traefik.docker.visiblewhennotrunning=true"
+```
+
+Controls whether containers that are not in "running" state (e.g., stopped, paused, exited) should still be visible to Traefik for service discovery.
+
+By default, Traefik only considers containers in "running" state.
+
```

Comment:
I suggest adding a bit more context:
```suggestion
When a container stops, its routers and services are removed from the configuration, resulting in 404 Not Found responses.

When this label is set to true, Traefik will:
- Keep the router and service configuration even when the container is not running
- Create services with empty backend server lists
- Return 503 Service Unavailable for requests to stopped containers (instead of 404 Not Found)
- Execute the full middleware chain, allowing middlewares to intercept requests
```
If you agree, please add it to the swarm provider documentation too.